src/lib/Behat/BrowserContext/ContentTreeContext.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\AdminUi\Behat\BrowserContext;
+
+use Behat\Behat\Context\Context;
+use Ibexa\AdminUi\Behat\Component\ContentTree;
+
+final class ContentTreeContext implements Context
+{
+    private ContentTree $contentTree;
+
+    public function __construct(
```

Comment:
```suggestion
    public function __construct(ContentTree $contentTree)
```
I think if we have only one parameter, we can write it in one line